src/ophys_etl/transforms/registry.py

```diff
@@ -0,0 +1,220 @@
+import boto3
+from typing import Optional, Union
+import time
+
+
+class RegistryConnection(object):
+    def __init__(
+            self,
+            table_name: str,
+            region: Optional[str] = "us-west-2",
+            aws_access_key_id: Optional[str] = None,
+            aws_secret_access_key: Optional[str] = None,
+            aws_session_token: Optional[str] = None):
+        """
+        Instantiate a connection to the dynamoDB table used as a model
+        registry.
+        
+        Parameters
+        ----------
+        table_name: str
+            The name of the DynamoDB table containing the registry
+        aws_access_key_id: Optional[str]
+            Access key id for AWS account. If not used, will discover
+            credentials according to the discovery order in boto3 (e.g. 
+            environment variables, ~/.aws/credentials file)
+            https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html    # noqa
+        aws_secret_access_key: Optional[str]
+            Secret access key for AWS account. See `aws_access_key_id`.
+        aws_session_token: Optional[str]
+            Session key for AWS account. Only needed when using temporary
+            credentials.
+
+        Methods
```

Comment:
remove this section if you're not going to fill it out?

Reply:
Oh oops, I was going to fill it out. Thanks!